maintainers/maintainer-list.nix

```diff
@@ -2407,6 +2407,12 @@
     githubId = 984691;
     name = "Evan Danaher";
   };
+  edbentley = {
+    email = "hello@edbentley.dev";
+    github = "edbentley";
+    githubId = 15923595;
+    name = "Ed Bentley";
```

Comment:
Please put this into a separate commit with the message: ``maintainers: add edbentley``